packages/notebook/src/widget.ts

```diff
@@ -232,6 +232,10 @@ export class StaticNotebook extends WindowedList {
     this._updateNotebookConfig();
     this._mimetypeService = options.mimeTypeService;
     this.renderingLayout = options.notebookConfig?.renderingLayout;
+
+    this._emptyNotebookMessage = new Widget();
+    this._emptyNotebookMessage.node.textContent =
+      'The notebook is empty. Click the + button on the toolbar to add a new cell.';
```

Comment:
Could you avoid creating the `Widget` in the constructor but only instantiate it when needed?
You should also wrap the text fort translation.